pallets/loans/src/lib.rs

```diff
@@ -1015,10 +1016,10 @@ impl<T: Config> Pallet<T> {
             .checked_sub(repay_amount)
             .ok_or(ArithmeticError::Underflow)?;
         let total_borrows = Self::total_borrows(currency_id);
-        let total_borrows_new = total_borrows
-            .checked_sub(repay_amount)
-            .ok_or(ArithmeticError::Underflow)?;
+        // TODO : total_borrows should contain interests
+        let total_borrows_new = total_borrows.saturating_sub(repay_amount);
```

Comment:
total_borrows < repay_amount and caused this issue:)

we need to find a way to make total_borrows contain also interests

Reply:
I think we can use this PR to fix #252. It only appears in the early days when total_borrows is too small.
When the case appears, the result of this modification is that `exchange_rate` won't be updated and no lender will get supply interest. That shouldn't be a problem.

Reply:
Could you modify the TODO to a NOTE and explain the code briefly? And pls add an edge test to catch this case.

Reply:
ok I'll do it